Shared/Server.swift

```diff
@@ -13,6 +13,8 @@ struct Server: Codable {
     let name: String
     let address: String
     let downloadDir: String?
+    let account: String?
```

Comment:
Is it possible to protect the RPC server only with username (without password)?
If not, i would suggest to encapsulate the account and password into a separate data structure.